scipy/interpolate/tests/test_interpolate.py

```diff
@@ -753,13 +753,21 @@ def test_read_only(self):
 
 
 class TestLagrange:
-
     def test_lagrange(self):
-        p = poly1d([5,2,1,4,3])
+        p = poly1d([5, 2, 1, 4, 3])
         xs = np.arange(len(p.coeffs))
         ys = p(xs)
-        pl = lagrange(xs,ys)
-        assert_array_almost_equal(p.coeffs,pl.coeffs)
+        pl = lagrange(xs, ys)
+        assert_array_almost_equal(p.coeffs, pl.coeffs)
```

Comment:
> Since you're adding a boolean flag, both values need to be tested.

Because there is a test case with the default parameter in the `def test_lagrange(self)`. Do I still need to add the `pl = lagrange(xs, ys, reduce=True)` explicitly?

Reply:
Ok, then please add a comment to this effect